lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -31,6 +35,16 @@
 
                 postRender: function() {
                     // Add subviews
+                    new CollectionListView({
+                        el: '.js-course-list',
+                        childView: CourseCardView,
+                        collection: new Backbone.Collection(this.context.course_codes),
```

Comment:
I tried to move the Collection instantiation to the initialize function, but if I put that initialization to before the `this.render();` statement, the collection does not render at all on manual testing.
If I move the collection instantiation after the `this.render();` statement, it defeat the purpose for other subviews to use the collection, right?
Can we leave this be and play with it later?